### PR TITLE
Create `Image` extension and `AppImage` enum

### PR DESCRIPTION
**Pull request by marinkman**
## What?

- Created `Image` extension
- Created `AppImage` enum and filled with `static let` images according to project assets.
- Created `ImageAppImageTests` class to tests `AppImage` enum

## Why?
According to issue #41 

## Anything Else?
Renamed the following assets and corresponding files:
   - `Exit_symbol_black@3x` to `Exit_symbol_black`
   - `Tick-mark_symbol_white` to `Tick_mark_symbol_white`
   - `General_background_image` to `General_background`
   - `Onboarding_background_image` to `Onboarding_background`
   - `Sleeping_astronaut_image` to `Sleeping_astronaut`

### CODE REVIEW

File: Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+extension Image {
+    /// The image used in the application.
+    enum AppImage {
+        /// An image of a white alarm symbol suitable for use in UI.
+        static let alarmSymbolWhite = Image("Alarm_symbol_white")
+        /// An image of a white bed symbol suitable for use in UI.
+        static let bedSymbolWhite = Image("Bed_symbol_white")
+        /// An image of a black exit symbol suitable for use in UI.
+        static let exitSymbolBlack = Image("Exit_symbol_black@3x")
```
Comment (dersim-davaod) [inline Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift:11]: Why this name has the `@3x` suffx?

File: Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        /// An onboarding background image suitable for use in UI.
+        static let onboardingBackgroundImage = Image("Onboarding_background_image")
+        /// A sleeping astronaut image suitable for use in UI.
+        static let sleepingAstronautImage = Image("Sleeping_astronaut_image")
+        /// An image of a statistic symbol suitable for use in UI.
+        static let statisticsSymbolPurple = Image("Statistics_symbol_purple")
+        /// An image of a colored sun symbol suitable for use in UI.
+        static let sunSymbolColored = Image("Sun_symbol_colored")
+        /// An image of a white symbol with a tick mark suitable for use in UI.
+        static let tickMarkSymbolWhite = Image("Tick-mark_symbol_white")
```
Comment (dersim-davaod) [inline Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift:31]: nit: can we be consistent when naming assets?
```suggestion
        static let tickMarkSymbolWhite = Image("Tick_mark_symbol_white")
```

**Review by dersim-davaod — COMMENTED** [review]
Few clarifications:

File: Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift (hunk 1 above)
Comment (marinkman) [reply Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift]: I didn't name the assets. I used the names that were given before.
I corrected this name.

File: Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+extension Image {
+    /// The image used in the application.
+    enum AppImage {
+        /// An image of a white alarm symbol suitable for use in UI.
```
[the remark this answers is not in the document]
Comment (marinkman) [reply Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift]: No :)
Fixed.

File: Sleep Tracker/Sleep TrackerUITests/Helpers/Image+AppImageTests.swift [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        XCTAssertEqual(moonWithStarsSymbolPurple, .AppImage.moonWithStarsSymbolPurple)
+        XCTAssertEqual(onboardingBackgroundImage, .AppImage.onboardingBackgroundImage)
+        XCTAssertEqual(sleepingAstronautImage, .AppImage.sleepingAstronautImage)
+        XCTAssertEqual(statisticsSymbolPurple, .AppImage.statisticsSymbolPurple)
+        XCTAssertEqual(sunSymbolColored, .AppImage.sunSymbolColored)
+        XCTAssertEqual(tickMarkSymbolWhite, .AppImage.tickMarkSymbolWhite)
+        XCTAssertEqual(viewSymbolPurple, .AppImage.viewSymbolPurple)
+    }
+
+    func testImageAppImageCanBeUsedInAFunctionWithAImageParameter() {
```
[the remark this answers is not in the document]
Comment (marinkman) [reply Sleep Tracker/Sleep TrackerUITests/Helpers/Image+AppImageTests.swift]: Fixed.

File: Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift (hunk 2 above)
Comment (marinkman) [reply Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift]: I corrected this name.

File: Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        /// An image of a purple home symbol.
+        static let homeSymbolPurple = Image("Home_symbol_purple")
+        /// An image of a purple moon symbol.
+        static let moonSymbolPurple = Image("Moon_symbol_purple")
+        /// An image of a colored moon symbol with stars.
+        static let moonWithStarsSymbolColored = Image("Moon_with_stars_symbol_colored")
+        /// An image of a purple moon symbol with stars.
+        static let moonWithStarsSymbolPurple = Image("Moon_with_stars_symbol_purple")
+        /// An onboarding background image.
+        static let onboardingBackgroundImage = Image("Onboarding_background_image")
```
Comment (dersim-davaod) [inline Sleep Tracker/Sleep Tracker/Helpers/Image+AppImage.swift:23]: nit: Why some images has the `Image` suffix and some are not?

which option is better for you from the perspective of client code:
```swift
foo(.AppImage.onboardingBackgroundImage)
```
or
```swift
foo(.AppImage.onboardingBackground)
```
?

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Review by meschya — APPROVED** [review]
LGMT

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Pull request merged**